test/deploy/kubernetes/runtime-pods/components_data_node3.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     metadata:
       labels:
         app: data-node3
+        cortx.io/machine-id: ccc8700fe6797ed532e311b0615c32a7
```

Comment:
Can we pull this value directly from pod metadata id? Instead of explicitly specifying it